### DISCUSSION TITLE
MixtureFinder

### DISCUSSION THREAD

**Discussion by josephhughes** [iqtree/iqtree2#330] [Q&A] [answered]
I'm running "IQ-TREE multicore version 2.3.6 for Linux x86 64-bit built Aug  1 2024" with the following command:
`/software/iqtree-v2.3.6/bin/iqtree2 -s test_aa.fa -m MIX+MF --prefix test`

following the documentation here: http://www.iqtree.org/doc/Complex-Models

> MixtureFinder is an approach to select the optimum number of classes and the substitution model in each class for a mixture model of Q matrices. Starting with version 2.3.1, MixtureFinder is available in IQ-TREE. To run MixtureFinder:
> 
> `iqtree -s example.phy -m MIX+MF`
> 

but I'm getting the following error message:

> 
> WARNING: Command-line argument `MIX+MF` differs from checkpoint `MIX`
> WARNING: Command-line differs from checkpoint!
> 

> For your convenience alignment with unique sequences printed to test.uniqueseq.phy
> ERROR: Error! The option -m '' can only work on DNA data set
> 

Is it really only possible to test mixture models for DNA alignments?

**Comment by roblanf** [comment, marked as answer]
Hi @josephhughes,

MixtureFinder only works with DNA models. This is explained in the paper: https://www.biorxiv.org/content/10.1101/2024.03.20.586035v2

There are lots of mixture models for Amino acids too, these are also available in IQ-TREE and explained in the docs under "Protein mixture models" here: http://www.iqtree.org/doc/Substitution-Models

Rob
